### ISSUE TITLE
Will this site ever move to gpodder.org?

### ISSUE THREAD

**Issue by adamvoss** [gpodder/gpodder.github.io#6]
[GitHub Pages supports custom domains](https://help.github.com/articles/using-a-custom-domain-with-github-pages/) so, if desired, it would be possible to server this website from gpodder.org once you no longer want to keep the extra redirect functionality currently provided by the gpodder.org host.

I ask mainly to know how judiciously to update links.

**Comment by thp** [comment]
Yes, we can think about this, but have to coordinate DNS switches.
